src/platform/OpenThread/GenericThreadStackManagerImpl_OpenThread.hpp

```diff
@@ -1353,7 +1353,7 @@ template <class ImplClass>
 void GenericThreadStackManagerImpl_OpenThread<ImplClass>::OnSrpClientStateChange(const otSockAddr * aServerSockAddr,
                                                                                  void * aContext)
 {
-    if (aServerSockAddr)
```

Comment:
I think this already checks the nullity of the pointer.

Reply:
Thanks for your correction, A null pointer is a special pointer value (usually 0) that indicates the pointer isn't pointing to any valid object.